### PR TITLE
Feat: Modal when the user is connected in a wrong network.

### LINKED ISSUE #127: Display a message when the user is in a wrong network
If the user is at .io the network should be `ropsten`, `goerli` or `mumbai`
If the user is at `.stg` or `.org` the network should be `mainnet` or `matic`

### PR DESCRIPTION
**Pull request by malaniz**
Feat: Modal when the user is connected in a wrong network.
img-7VGumr_000

Closes #127

### CODE REVIEW

File: src/modules/translation/defaults/en.json [hunk 1]
```diff
@@ -31,6 +31,10 @@
         "builder": "Builder",
         "events": "Events",
         "dao": "DAO"
+      },
+      "wrongNetwork": {
+        "header": "Wrong Network",
+        "message": "You need to be connected to {expectedChainName} Network to use this app, but you are currently connected to {currentChainName} Network."
```
Comment (nicosantangelo) [inline src/modules/translation/defaults/en.json:37]:
```suggestion
        "message": "You need to be connected to the {expectedChainName} Network to use this app, but you are currently connected to {currentChainName}."
```

File: src/containers/Navbar/Navbar.types.tsx [hunk 2]
```diff
@@ -1,12 +1,26 @@
+import { ChainId } from '@dcl/schemas'
 import { NavbarProps as NavbarComponentProps } from 'decentraland-ui'
 
+export declare type WrongNetworkModalI18N = {
```
Comment (nicosantangelo) [inline src/containers/Navbar/Navbar.types.tsx:4]:
```suggestion
export type WrongNetworkModalI18N = {
```

**Review by nicosantangelo — CHANGES_REQUESTED** [review]
I understand that we're using the base components from decentraland-UI but we still need to adjust margins and paddings a bit

image

**Review by nachomazzara — APPROVED** [review]

**Pull request merged**